### PR TITLE
Update README.md wrt BMFF

### PR DESCRIPTION
**Pull request by postscript-dev**
Suggested during discussion in https://github.com/Exiv2/exiv2/issues/2398.

### CODE REVIEW

**Review by 1div0 — CHANGES_REQUESTED** [review]
I would use full name JPEG XL instead of the abbreviation.

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The file ReadMe.txt in a build bundle describes how to install the library on th
     - [Library Initialisation and Cleanup](#InitAndCleanup)
     - [Cross Platform Build and Test on Linux for MinGW](#CrossPlatformSupport)
     - [Static and Shared Libraries](#StaticShared)
-    - [Support for bmff files (CR3, HEIF, HEIC, and AVIF)](#BMFF)
+    - [Support for bmff files (e.g., AVIF, CR3, HEIF, HEIC, and JXL)](#BMFF)
```
Comment (kmilos) [inline README.md:52]: No comma after "e.g."?

File: README.md [hunk 2]
```diff
@@ -898,7 +898,7 @@ This is discussed: [https://github.com/Exiv2/exiv2/issues/1230](https://github.c
 [TOC](#TOC)
 <div id="BMFF">
 
-## Support for bmff files (CR3, HEIF, HEIC, and AVIF)
+## Support for bmff files (e.g., AVIF, CR3, HEIF, HEIC, and JXL)
```
Comment (kmilos) [inline README.md:901]: Ditto?

**Comment by postscript-dev** [comment]
@1div0 
> I would use full name JPEG XL instead of the abbreviation.

I have made the change and will try and use the full name in the future.

**Review by kmilos — APPROVED** [review]

**Review by 1div0 — APPROVED** [review]

**Pull request merged**